ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/accordion.html

```diff
@@ -34,31 +34,31 @@
      <div data-sly-call="${questionMark.questionMark @componentId=accordion.id, longDescription=accordion.description, bemBlock='cmp-accordion'}" data-sly-unwrap></div>
      <div data-sly-call="${shortDescription.shortDescription @componentId=accordion.id, shortDescriptionVisible=accordion.tooltipVisible, shortDescription=accordion.tooltip, bemBlock='cmp-accordion'}" data-sly-unwrap></div>
      <div data-sly-call="${longDescription.longDescription @componentId=accordion.id, longDescription=accordion.description, bemBlock='cmp-accordion'}" data-sly-unwrap></div>
-    
+
     <div data-sly-test="${wcmaccordion.items.size > 0}"
```

Comment:
lets remove the dependency on wcmaccordion altogether.
`data-sly-test="${accordion.exportedItems.size > 0}"`

Reply:
Sure, done that